openwisp_radius/migrations/0002_initial_openwisp_radius.py

```diff
@@ -0,0 +1,241 @@
+# Generated by Django 2.1.1(2018-09-27) + Manually Edited
+
+import re
+import uuid
+
+import django.core.validators
+import django.db.models.deletion
+import django.utils.timezone
+import model_utils.fields
+from django.conf import settings
+from django.db import migrations, models
+from django_freeradius.migrations import add_default_group_to_existing_users, add_default_groups
+from openwisp_radius.migrations import add_default_organization
+
+import openwisp_radius.models
+import openwisp_users.mixins
+
+class Migration(migrations.Migration):
+
```

Comment:
i'd add a docstring here explaining this adds the custom logic of openwisp-radius